### ISSUE TITLE
Unable to restore (remove ntrboot flash from) my Acekard 2i HW81

### ISSUE THREAD

**Issue by Machow8** [ntrteam/ds_ntrboot_flasher#41]
Hello,

I could flash ntrboot hax in my Acekard 2i hw81 via DS Lite, but, 3DS' ntrboot flasher program says that flashcart isn't supported.

On log file there is:

[AK2i]: HW Revision = 34b8b7ff
[AK2i]: Shutdown

Is there a way to solve this in a new version of your program?

Thanks.

**Comment by Antviau** [comment]
Hi, use decript9wip it's the only thing that works in those kind of cases for some reason.

**Comment by Antviau** [comment]
And I now that this is 3 years old, but someone had to answer it, I guess.

**Comment by ghost** [comment]
Thanks! No matter how long it takes for an answer like yours come, it's always welcome to find alternatives like that! =]

I'll check if this works for my case.

**Comment by Antviau** [comment]
No, problem, I had the same issue and sadly had to figure that one on my
own, that really is a weird issue though.

**Comment by Exayl342** [comment]
I have the same problem but i use r4i rts 3ds card
